docs/pages/access-controls/access-request-plugins/ssh-approval-msteams.mdx

```diff
@@ -33,51 +33,7 @@ in your Teleport cluster.
 
 ## Step 2/9. Install the Teleport Microsoft Teams plugin
 
```

Comment:
ms teams doesn't have the helm install.  We should either add the helm install or retain the current version.

Reply:
Would you mind clarifying what you mean by "the helm install"? I see that there's a Helm chart available for the Microsoft Teams plugin, if that's what you're referring to: https://github.com/gravitational/teleport-plugins/tree/master/charts/access/msteams